src/main/kotlin/org/jetbrains/research/testspark/uiUtils/GenerateTestsTabHelper.kt

```diff
@@ -14,7 +14,7 @@ object GenerateTestsTabHelper {
      * @return the editor corresponding to the test case, or null if it does not exist
      */
     fun getEditorTextField(testCaseName: String, generatedTestsTabData: GeneratedTestsTabData): EditorTextField? {
-        val middlePanelComponent = generatedTestsTabData.testCasePanels[testCaseName]?.getComponent(2) ?: return null
+        val middlePanelComponent = generatedTestsTabData.testCaseNameToPanels[testCaseName]?.getComponent(2) ?: return null
```

Comment:
What resides under `getComponent(2)`? it is better to comment on it or move into a named constant.

Reply:
@arksap2002